### PR TITLE
Set max buyer's security deposit to 20% instead of 10%

### PR DESCRIPTION
**Pull request by ManfredKarrer**
A Bisq trader suggested that change to cover risks with volatility
for seller.

### CODE REVIEW

File: core/src/main/java/bisq/core/btc/wallet/Restrictions.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public static double getMinBuyerSecurityDepositAsPercent() {
     }
 
     public static double getMaxBuyerSecurityDepositAsPercent() {
-        return 0.1; // 10% of trade amount. For a 1 BTC trade it is about 400 USD @ 4000 USD/BTC
+        return 0.2; // 20% of trade amount. For a 1 BTC trade it is about 400 USD @ 4000 USD/BTC
```
Comment (devinbileck) [inline core/src/main/java/bisq/core/btc/wallet/Restrictions.java]: The USD amount in the comment is now incorrect.

> 400 USD @ 4000 USD/BTC

Should be

> 800 USD @ 4000 USD/BTC

**Review by devinbileck — APPROVED** [review]
utACK
Just a minor nit about the comment.

**Comment by ripcurlx** [comment]
@ManfredKarrer Sorry for taking over your commit 😉 . I just made the change @devinbileck was requesting and didn't want to create another commit message.

**Review by ripcurlx — APPROVED** [review]
ACK

**Pull request merged**